qreu/email.py

```diff
@@ -270,19 +270,17 @@ def add_body_text(self, body_plain=False, body_html=False):
         self.email.attach(msg_part)
         return True
 
-    def add_attachment(self, input_buff=False, input_b64=False, attname=False):
+    def add_attachment(self, input_buff=False, attname=False):
```

Comment:
Input buffer must be required as it is the attachment itself.
```suggestion
    def add_attachment(self, input_buff=False, attname=False):
    def add_attachment(self, input_buff, attname=False):
```